catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -76,16 +78,31 @@ def parse_gh_date(d) -> datetime.datetime:
     return datetime.datetime.fromisoformat(d.rstrip("Z"))
 
 
-def get_urgency_if_urgent(pr: dict) -> ReviewDelta | None:
-    updated_at = parse_gh_date(pr["updated_at"])
+def get_urgency_if_urgent(gh: GitHubAPI, pr: dict) -> ReviewDelta | None:
+    events = gh.get_issue_events(base_repo_name(pr), pr["number"])
+    ready_for_review_date = pr["created_at"]
+    for event in reversed(events):
+        # Find the latest "ready_for_review" event.
+        # We don't need to check for or stop if we
+        # find a draft event because drafted PRs are
+        # already filtered out at this point. Any draft
+        # events present in the events list would
+        # be further back in time than the latest
+        # (and therefore relevant) ready_for_review
+        # event will appear before we reach any
+        # draft events.
+        if event["event"] == "ready_for_review":
+            ready_for_review_date = event["created_at"]
```

Comment:
If there are multiple `ready_for_review` events in a PR's event timeline, this seems like it'll get the oldest event - does it make sense to `break` on the case where a ready for review event is found?

Reply:
It 100000% should have a break, thank you for catching that!